src/Okta.Sdk/Internal/HttpRequestMessageProvider.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+
+namespace Okta.Sdk.Internal
+{
+    public class HttpRequestMessageProvider : IHttpRequestMessageProvider
+    {
+        private readonly Dictionary<HttpVerb, HttpMethod> _httpMethods = new Dictionary<HttpVerb, HttpMethod>
+        {
+            { HttpVerb.Get, HttpMethod.Get },
+            { HttpVerb.Post, HttpMethod.Post },
+            { HttpVerb.Put, HttpMethod.Put },
+            { HttpVerb.Delete, HttpMethod.Delete },
+        };
+
+        public static IHttpRequestMessageProvider Default => new HttpRequestMessageProvider();
+
+        public HttpRequestMessage CreateHttpRequestMessage(HttpRequest httpRequest, string relativePath)
```

Comment:
Nit: Let's add comments for public methods.

Reply:
Good catch 👍